app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,29 @@
+class WelcomeController < ApplicationController
+
+  before_action :find_top_ten_movies, only: :index
+  before_action :find_top_ten_books, only: :index
+  before_action :find_top_ten_albums, only: :index
+  before_action :find_spotlight_work, only: :index
```

Comment:
If you're only using it for one action, is it worth making it a before action? Why not just put it in the index action? I know we taught to use the pattern of a before_action, but you can also execute your private methods by just calling them in the `index` method too ;)